aixplain/factories/agent_factory/__init__.py

```diff
@@ -67,6 +67,8 @@ def create(
         Returns:
             Agent: created Agent
         """
+        # assert that at least one tool is provided
+        assert len(tools) > 0, "At least one tool must be provided."
```

Comment:
Do we need to force user to have at least one `ModelTool `or `PipelineTool`? I think if the user wants to use only serp api search we should let it do that.